direct/io-s3/src/main/java/cz/o2/proxima/direct/s3/S3Client.java

```diff
@@ -45,6 +53,9 @@
 @Slf4j
 class S3Client implements Serializable {
 
+  /** Part size in multi-part upload (5MB). */
+  private static final int UPLOAD_PART_SIZE = 5 * 1024 * 1024;
```

Comment:
Should this be configurable?

Reply:
not in this iteration, this is a reasonable default (the same is used by native s3 client) and I don't think it needs to be override in most real world scenarios. We can always change this once we have a use case.